strings/owo_talk.json

```diff
@@ -0,0 +1,76 @@
+{
+	
+    "wowds": {
+        "ID": "furball",
+        "fuck": "OwO",
+        "shit": "uwu",
+        "dick": "bulge",
+        "weed": "catnip",
+        "arrested": "nuzzled",
+        "arrest": "pounce",
+        "virologist": "good kitty",
+        "viro": ["kittie","neko"],
+        "virology": ["kittie","neko"],
+        "sec":"battle cats",
+        "security":"battle cats",
+        "shitsec":"bad battle cats",
+        "shitcurity":"bad battle cats",
+        "captain": "catniptain",
+        "cap": "cat",
+        "felinid": ["kittie","neko"],
+        "feline": ["kittie","neko"],
+        "catperson": ["kittie","neko"],
+        "person": "cat",
+        "lizard": "scalie",
+        "damn": "mew",
+        "kill": "yiff",
+        "killed": "yiffed",
+        "murder": "yiff",
+        "murdered": "yiffed",
+        "thief": "stealer",
+        "bad": "naughty",
+        "horrible": "naughty",
+        "chocolate": ["pickle","cucumber","water"],
+        "coco": ["pickle","cucumber"],
+        "choko": ["pickle","cucumber"],
+        "cocoa": ["pickle","cucumber"],
+        "heal": "lickie",
+        "god": "the biggest kitty",
+        "gods": "the biggest kitty",
+        "scientist": "smart kitty",
+        "engineer": "handy kitty",
+        "hop": "head kitty",
+        "centcom": "daddy-cat",
+        "centcomm": "daddy-cat",
+        "syndicate": "doggy",
+        "lrp": "kitty is dum-dum",
+        "medbay":"cat nest",
+        "med":"mama-cat",
+        "doctor":"mama-cat",
+        "cmo":"mama-cat",
+        "healer":"mama-cat",
+        "gun":"claw",
+        "guns":"claws",
+        "wizard":"magic kitty",
+        "toolbox":"litterbox",
+        "QM":"Quarter Kitty",
+        "cargo":"box",
+        "good":"nya",
+        "sci":"smart kitties",
+        "RD":"smartest cat",
+        "head of":"head kitty of",
+        "nyagger":"best kitten",
+        "catbeast":"kitten",
+        "catbeasts":"kittens",
+        "I":"cat",
+        "me":"kitten"
```

Comment:
remove the "me":"kitten" translation or it makes "meow" appear as "kittenow"

Reply:
thanks, done